src/otaclient_iot_logging_server/aws_iot_logger.py

```diff
@@ -68,40 +73,45 @@ def __init__(
 
         self._session_config = session_config
         self._log_group_name = session_config.aws_cloudwatch_log_group
+        self._metrics_group_name = session_config.aws_cloudwatch_metrics_log_group
         self._interval = interval
```

Comment:
About the variable naming here, previously we only have one log_group, so log_group equals to otaclient_logs log_group.
But now we have two log_groups, it becomes kind of ambiguous, `(otaclient_logs_)log_group` and `(otaclient_)metrics_group` are both `aws_cloudwatch_log_group`.

I suggest that we prefix a `otaclient_logs` to the variable name related to `(otaclient_logs_)log_group`

Reply:
Thank you, agree.
Long formalized clear name should be better than short unclear name, fixed in the latest commit.